File: GXpert/GXpert.Web/Modules/ServerTypes/Membership/SignUpRequest.ts
import { ServiceRequest } from "@serenity-is/corelib";

export interface SignUpRequest extends ServiceRequest {
    DisplayName?: string;
    Mobile?: string;
    Email?: string;
    Password?: string;
    FirstName?: string;
    LastName?: string;
    PrnNo?: string;
    Recaptcha?: string;
    ExternalProviderToken?: string;
}

File: GXpert/GXpert.Web/Modules/Membership/Account/SignUp/SignUpRequest.cs

namespace GXpert.Membership;

public class SignUpRequest : ServiceRequest
{
    public string DisplayName { get; set; }
    public string Mobile { get; set; }
    public string Email { get; set; }
    public string Password { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string PrnNo { get; set; }
    public string Recaptcha { get; set; }
    public string ExternalProviderToken { get; set; }
}